libafl_bolts/src/lib.rs

```diff
@@ -451,6 +451,13 @@ impl Error {
     {
         Error::InvalidCorpus(arg.into(), ErrorBacktrace::new())
     }
+
+    /// If [`Feedback::last_result`] is called before the `Feedback` is actually run.
+    #[cfg(feature = "track_hit_feedbacks")]
+    #[must_use]
+    pub fn premature_last_result() -> Self {
```

Comment:
This doesn't belong in bolts, IMHO. I would move it into feebacks:: or otherwise make it a method on the Feeback trait